src/tests/ftest/io/dfuse_find_cmd.py

```diff
@@ -0,0 +1,215 @@
+                    challenger_path, samples, challenger_dir_trees)
+
+        except CommandFailure as error:
+            self.log.error("FindCmd Test Failed: %s", str(error))
+            self.fail("Test was expected to pass but it failed.\n")
+        except Exception as error:
+            self.log.error("FindCmd Test Failed: %s", str(error))
+            raise
+        finally:
+            self._teardown_dfuse(self._dfuses)
+            self.destroy_containers(self._containers)
+            self.pool.destroy()
+
+            if challenger and challenger_path:
+                shutil.rmtree(challenger_path, ignore_errors=True)
+
+        self.log.info("DAOS Stats")
+        daos_stats.print_stats()
+
+        if challenger:
+            self.log.info("Challenger Stats")
+            challenger_stats.print_stats()
+
+            self._compare_results(daos_stats, challenger_stats, "all_files")
+            self._compare_results(daos_stats, challenger_stats, "same_suffix")
+            self._compare_results(daos_stats, challenger_stats, "unique_file")
+
+    def _compare_results(self, daos_stats, challenger_stats, tag):
+        daos_max_time, _, _ = daos_stats.get_stat(tag)
+        challenger_max_time, _, _ = challenger_stats.get_stat(tag)
+
+        if daos_max_time >= challenger_max_time:
+            self.log.warning(
+                "Impossible, DAOS is slower running '{0}' tag".format(tag))
+        else:
+            self.log.info(
+                "DAOS is equal or faster running '{0}' tag".format(tag))
+
+    def _run_commands(self, test_path, samples, dir_trees):
+
+        profiler = general_utils.SimpleProfiler()
+        profiler.set_logger(self.log.info)
+
+        self.log.info("Sampling path: {0}".format(test_path))
+
+        for i in range(samples):
+            self.log.info(
+                "Running sample number {0} of {1}".format(i + 1, samples))
+            profiler.run(
+                self._run_cmd,
+                "all_files",
+                u"find {0} -name *.needle".format(test_path))
+
+            number = random.randrange(self._needles - 1)
+            file_name = "*_{:05d}.needle".format(number)
+            profiler.run(self._run_cmd, "same_suffix",
+                         u"find {0} -name {1}".format(test_path, file_name))
+
+            dir_tree = random.choice(dir_trees)
+            file_name, _ = dir_tree.get_probe()
+            profiler.run(self._run_cmd, "unique_file",
+                         u"find {0} -name {1}".format(test_path, file_name))
+
+        return profiler
+
+    def _setup_containers(self, dfs_path, cont_count):
+        mount_dirs = []
+        for count in range(cont_count):
+            self.add_container(self.pool)
+            cont_dir = "{}_daos_dfuse_{}".format(self.pool.uuid, count)
+            mount_dir = os.path.join(dfs_path, cont_dir)
+            self.log.info(
+                "Creating container Pool UUID: {0} Con UUID: {1}".format(
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments